<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FanControlDialog</class>
 <widget class="QDialog" name="FanControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="fanIndexLabel">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>48</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1#</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>风机控制</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>运行状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>故障状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>报警状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>过流报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>过压报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>过热报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>65</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>过力矩报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>风机状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="runStateLabel">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>180</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="faultStateLabel">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>180</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="alertStateLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ofStateLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ovStateLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ohStateLabel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="otStateLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>130</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>运行频率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>190</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输出电压</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>190</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输出电流</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>190</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输出功率</string>
   </property>
  </widget>
  <widget class="QLabel" name="runFreqLabel">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>180</y>
     <width>30</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ovLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>30</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ocLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>220</y>
     <width>30</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="opLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>220</y>
     <width>30</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>180</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>220</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>KW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>80</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>风机控制选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>25</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>手动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>25</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>自动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>110</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>点击切换手自动模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>80</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>风机手动控制</string>
   </property>
  </widget>
  <widget class="MyCustomSlider" name="outputFreqSlider">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="fanOpenPushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>风机运行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fanStopPushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>风机停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="switchFanModePushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="fanIndexComboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>266</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输出频率选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前频率</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputFreqLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>180</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>323</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>运行流量</string>
   </property>
  </widget>
  <widget class="QLabel" name="airInputLabel">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>180</y>
     <width>30</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>180</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>气源采集电磁阀开启时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="valveCloseTimeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="valveOpenTimeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>60</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>180</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>气源电磁阀间隔时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>秒</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ValveOpenPushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>气源阀开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ValveClosePushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>气源阀关</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>420</y>
     <width>120</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>气源电磁阀状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="airValveStateLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCustomSlider</class>
   <extends>QSlider</extends>
   <header location="global">mycustomslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
